src/main/java/org/ibp/api/java/impl/middleware/inventory/manager/validator/LotInputValidator.java

```diff
@@ -133,7 +164,32 @@ private void validateTransactionStatus(final List<ExtendedLotDto> lotDtos, final
 
 		if (transactionDtos.stream().map(TransactionDto::getTransactionStatus)
 			.anyMatch(s -> s.equals(TransactionStatus.CONFIRMED.getValue()))
-			&& (updateRequestDto.getUnitId() != null)) {
+			&& (lotSingleUpdateRequestDto.getUnitId() != null)) {
+
+			this.errors.reject("lots.transactions.status.confirmed.cannot.change.unit");
+		}
+	}
+
+	private void validateTransactionStatus(final LotMultiUpdateRequestDto lotMultiUpdateRequestDto) {
```

Comment:
I will try to make this validation more generic, with a parameter List<String> lotUUID and name validateNoConfirmedTransactions and unify both functions